organization/external-communication.md

```diff
@@ -0,0 +1,168 @@
+
+### How to add a new blog post
+
+Currently, there are several groups of people who submit blog posts to the pyOpenSci blog: pyOpenSci staff, package authors/maintainers of [pyOpenSci packages](https://www.pyopensci.org/python-packages.html) and pyOpenSci community members and friends who have participated in a pyOpenSci event. If you are interested in submitting a blog post to pyOpenSci, please reach out to us at [media@pyopensci.org](mailto:media@pyopensci.org) to discuss your post prior to writing it and submitting a PR.
+
+**How to submit a blog post to pyOpenSci**
+
+The pyOpenSci blog post submission process defined below should be used by everyone submitting a blog post.
+1. Fork the pyOpenSci website GitHub repository: [https://github.com/pyOpenSci/pyopensci.github.io](https://github.com/pyOpenSci/pyopensci.github.io).
+2. Work on your blog post locally. Be sure to include alt text for all images submitted, and create an author page if you do not already have one.
+
+```
+```{figure} filepath/to/your/image.png
+:scale: 100 %
+:alt: This text will appear as the alt text to your image.
+
+This text will appear as the caption to your image.```
+```
+
+To create an author page, submit a pull request from your fork of the pyOpenSci website repository, with changes to the [authors.yml](https://github.com/pyOpenSci/pyopensci.github.io/blob/main/_data/authors.yml) file. If you would like to include a headshot, you can add it to the [`people` directory in the `images` directory.](https://github.com/pyOpenSci/pyopensci.github.io/tree/main/images/people)
+
+3. Submit a pull request from your fork of the website repo to [https://github.com/pyOpenSci/pyopensci.github.io](https://github.com/pyOpenSci/pyopensci.github.io).
+4. Make sure that the blog post passes CI checks that look for spelling errors, broken links and more. Please note that occasionally the `htmlproofer` check will fail because the URL does not exist yet. We have implemented a fix for this, but it occasionally does not work.
+5. Request review of the blog post. A member of the pyOpenSci staff should always review any new content being added to the website.
+6. Once you have an approving review, the pull request can be merged. In the case of guest blog posts, a member of pyOpenSci's staff or contributor team will merge your post.
+
+pyOpenSci has GitHub organization-wide policies in place that require all pull requests are reviewed before being merged. In some instances, we may bypass those if the PR is a small fix or critical update. In most cases a review is preferred.
+
+:::{admonition} Yaml elements required for blog posts
+:class: tip
+
+All package authors, maintainers (and users) of a pyOpenSci-accepted package are welcome and encourage to submit a blog post (or series of blog posts) about their review experience and/or the use of the package. When submitting your blog post, please include the following YAML elements:
+
+```
+layout: single
```

Comment:
I just added a last_modified yaml element and a script (that is not in our ci yet) that updates the last updated date for any blog post that we add (or event). It just needs to have last_modified as a key in the yaml. 

it's a bash script. so in theory if someone runs that locally the last_updated key will be updated and the bottom of the page last_updated element will also get updated!

please add that key here and add a myst note about the bash script here as well. 

https://github.com/pyOpenSci/pyopensci.github.io/blob/main/scripts/date-updated.sh

there are instructions for running it at the top of the script.